source/camera/advanced/capture_hdr_print_normals.py

```diff
@@ -0,0 +1,51 @@
+"""
+Capture Zivid point clouds, compute normals and print a subset.
+
+For scenes with high dynamic range we combine multiple acquisitions to get an HDR point cloud.
+"""
+
+from typing import Any
+
+import zivid
+from nptyping import NDArray
+
+
+def _print_normals(radius: int, normals: NDArray[Any, Any, 3]):
+    line_sep = "-" * 50
+    num_of_rows = normals.shape[0]
+    num_of_cols = normals.shape[1]
+    print(line_sep)
+    for row in range(int(num_of_rows / 2 - radius), int(num_of_rows / 2 + radius)):
+        for col in range(int(num_of_cols / 2 - radius), int(num_of_cols / 2 + radius)):
+            normal_string = f"Normals ({row}, {col}): ["
+            for i in range(normals.shape[2]):
+                normal_string += f"{normals[row, col, i]:.3} ".rjust(10)
+            normal_string += "]"
+            print(normal_string)
+    print(line_sep)
+
+
+def _main():
+    app = zivid.Application()
+
+    print("Connecting to camera")
+    camera = app.connect_camera()
+
+    print("Configuring settings")
+    settings = zivid.Settings(acquisitions=[zivid.Settings.Acquisition(aperture=fnum) for fnum in (11.31, 5.66, 2.83)])
+
+    print("Capturing frame (HDR)")
+    with camera.capture(settings) as frame:
+        point_cloud = frame.point_cloud()
+
+        print("Computing normals and copying them to CPU memory")
+        normals = point_cloud.copy_data("normals")
+
+        print("Printing normals for the central ")
+        radius_of_pixels_to_print = 5
+        print(f"{radius_of_pixels_to_print * 2} x {radius_of_pixels_to_print * 2} pixels")
```

Comment:
it is a bit weird to have` radius_of_pixels_to_print = 5` between two prints. I would go for:

```
radius_of_pixels_to_print = 5
print(f"Printing normals for the central {radius_of_pixels_to_print * 2} x {radius_of_pixels_to_print * 2} pixels")
```

Reply:
done, but the formatter will split the print